src/test/java/com/courtcanva/ccfranchise/repositories/OrderRepositoryTest.java

```diff
@@ -29,11 +31,14 @@ class OrderRepositoryTest {
     @Autowired
     private OrderRepository orderRepository;
 
+
     @BeforeEach
     void setOrderRepositoryUp() {
         orderRepository.deleteAll();
-        staffRepository.deleteAll();
         franchiseeRepository.deleteAll();
+        orderRepository.save(OrderTestHelper.Order1());
+        orderRepository.save(OrderTestHelper.Order2());
+        staffRepository.deleteAll();
```

Comment:
It's better to leave `staffRepository.deleteAll()` above  `franchiseeRepository.deleteAll();` because `staff` depend on `franchisee`